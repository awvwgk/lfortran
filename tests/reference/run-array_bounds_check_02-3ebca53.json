{
    "basename": "run-array_bounds_check_02-3ebca53",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_02.f90",
    "infile_hash": "eaa31b185f0a51c3937f0238d112eb0fc90b3e410e59da7fe194c389",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_02-3ebca53.stderr",
    "stderr_hash": "2af8a2df076cac41a76f51056e46d55888189018c4c56a05a7ff98b3",
    "returncode": 1
}