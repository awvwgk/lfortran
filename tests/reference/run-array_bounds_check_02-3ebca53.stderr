At 11:9 of file tests/errors/array_bounds_check_02.f90
Runtime Error: Size mismatch in assignment to 'y'

LHS size is 4 and RHS size is 3
